x-pack/plugins/secops/public/components/page/hosts/uncommon_process_table/index.mock.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { UncommonProcessesData } from 'x-pack/plugins/secops/common/graphql/types';
+import { UncommonProcessesData } from '../../../../../common/graphql/types';
```

Comment:
note: I changed this to not use the `x-pack` base paths. We are mixing relative with base pathing. We should be consistent. Since relative pathing is the norm, I changed the areas we are pulling in base path to be relative pathing.